frontend/src/metabase/query_builder/actions.js

```diff
@@ -480,13 +481,17 @@ export const SET_CARD_AND_RUN = "metabase/qb/SET_CARD_AND_RUN";
 export const setCardAndRun = createThunkAction(SET_CARD_AND_RUN, (runCard, shouldUpdateUrl = true) => {
```

Comment:
Yep I can clean it up, I've been using `nextCard` in one other part of this PR which probably makes more sense